content/docs/reference/certificates.mdx

```diff
@@ -152,52 +151,10 @@ Kubernetes does not support `certificate_authority`
 </TabItem>
 </Tabs>
 
-## Client Certificate Authority {#client-certificate-authority}
-
 :::caution
 
-This setting is deprecated; it has been moved to a new **Downstream mTLS Settings** group (see the [**Certificate Authority**](/docs/reference/downstream-mtls-settings#ca) setting there).
+We've removed support for the `client_ca` and `client_ca_file` settings. To configure client certificates for downstream connections, see the **Downstream mTLS Settings** group (see the [**Certificate Authority**](/docs/reference/downstream-mtls-settings#ca) setting there).
```

Comment:
I think it would be fine to remove this note entirely. I'm not sure it adds much.

If we do keep it, I would re-phrase this: I think it would be more accurate to say this setting has "moved".